web/eslint.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "modules": true
+        }
+    },
+    "extends": "eslint:recommended",
```

Comment:
It must have change lately. Do you have a project where these rules are up to date?